src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -98,7 +98,11 @@ static void handle_signal(int signum)
 {
   assert(signum == SIGINT || signum == SIGTERM);
   derr << "*** Got signal " << sig_str(signum) << " ***" << dendl;
-  ioctl(nbd, NBD_DISCONNECT);
+  dout(20) << __func__ << ": " << "sending NBD_DISCONNECT" << dendl;
+  if (ioctl(nbd, NBD_DISCONNECT) < 0)
+    cerr << "rbd-nbd: the device is not used" << std::endl;  
```

Comment:
For curly brackets, I also like it for one line blocks, but I find it is not used in rbd-nbd.cc